ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/hosts.ini

```diff
@@ -67,6 +67,7 @@ mpls
 none_configuration
 ntp
 patch-panel
+peer_filters
```

Comment:
we use `-` in hostnames for molecule. Please update files (clean up the ones with `_`)
this new "host" should also be added in `hosts.ini` in 4.0 molecule.